cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -33,6 +33,10 @@ cros_gralloc_driver::~cros_gralloc_driver()
 	if (drv_kms_) {
 		int fd = drv_get_fd(drv_kms_);
 		drv_destroy(drv_kms_);
+		if (!is_kmsro_enabled())
+		{
```

Comment:
The left brace of a code block inside a function definition shall be put on the same row.

Reply:
yes, you are right, i forget to run the clang-format before submit patch :(